kernel/drivers/ps2.rs

```diff
@@ -46,6 +51,7 @@ impl Ps2 {
             mouse_i: 0,
             mouse_x: 0,
             mouse_y: 0,
+            layout: layouts::Layout::FRENCH,
```

Comment:
The default is still set to french. Perhaps a #[cfg(language)] could be used somehow?